src/parameter_declarations.ts

```diff
@@ -147,8 +148,7 @@ Parameter.declare({
 Parameter.declare({
 	name: DriveIdParameter,
 	aliases: ['-d', '--drive-id'],
-	description: 'the ArFS entity ID associated with the target drive',
-	required: true
```

Comment:
How much functionality does this cause us to lose in all the functions where a driveID is required? Could we not just make manifests pertain to ONLY folders, and have users use a rootFolderId rather than a driveId?

Reply:
All of the methods that have a required driveId or folderId seem to be using `getRequiredParameterValue` appropriately. I was under the impression that these `required` fields were redundant here now.

As for only using folderId, I guess that would make sense since we don't ever allow a user to upload to a driveId -- we have them upload to their rootFolderId. 

The original idea here was to be able to create a manifest of a drive, which is why we were using driveId as a parameter. I can switch it up, that will also simplify the API in Core